src/main/java/com/cburch/logisim/std/ttl/Ttl7432.java

```diff
@@ -26,12 +25,9 @@
   private static class OrGateHDLGeneratorFactory extends AbstractGateHDLGenerator {
 
     @Override
-    public ArrayList<String> getLogicFunction(int index) {
-      final var contents = new ArrayList<String>();
-      contents.add("   " + Hdl.assignPreamble() + "gate_" + index + "_O" + Hdl.assignOperator()
-              + "gate_" + index + "_A" + Hdl.orOperator() + "gate_" + index + "B;");
-      contents.add("");
-      return contents;
+    public LineBuffer getLogicFunction(int index) {
+      return LineBuffer.getHdlBuffer()
```

Comment:
BTW: would it be easier if HDL placeholders were always available (so just `getBuffer()` and you all set)?

Reply:
No, I actually like the differentiation between ```getBuffer```, and  ```getVhdlBuffer``` as it (1) makes LineBuffer more versatile, and (2) it is more readable as in the second case we know we are going to write HDL code :-)